### ISSUE TITLE
[Feature Request] Translate to EN

### ISSUE THREAD

**Issue by VitorGultzgoff** [ljianshu/Blog#64]
I'm afraid of the answer to what I'm going to request for the community here, but let's do it:

Looking into the _huge content_ stored and discussed here, I realized that there is so much **high-quality advanced WEB & JS content** in Chinese which kind of limits the reach of the content.

Absolutely the Google translate and other tools help a lot to understand the context in general cases, but we're dealing with advanced terms and concepts, those little details really matter.

What about we **translate** the project, would be a bad idea?

**Comment by ljianshu** [comment]
Thanks! It's my honor to hear that my blog will be translated to English.Can we talk about it on WeChat? My WeChat is 'renxinsidong'

**Comment by VitorGultzgoff** [comment]
> Thanks! It's my honor to hear that my blog will be translated to English.Can we talk about it on WeChat? My WeChat is 'renxinsidong'
Absolutely, actually I've never used this platform but won't be a problem, I'll call you, thanks a million for the attention.

**Comment by VitorGultzgoff** [comment]
> Thanks! It's my honor to hear that my blog will be translated to English.Can we talk about it on WeChat? My WeChat is 'renxinsidong'

Absolutely, actually I've never used this platform but won't be a problem, I'll call you, thanks a million for the attention.

**Comment by ljianshu** [comment]
> > Thanks! It's my honor to hear that my blog will be translated to English.Can we talk about it on WeChat? My WeChat is 'renxinsidong'
> 
> Absolutely, actually I've never used this platform but won't be a problem, I'll call you, thanks a million for the attention.

sorry. What about Facebook or Twitter?
